### PR TITLE
Initial table of contents design

### LINKED ISSUE #34: Style and implement table of contents
The table of contents needs to be modified to accommodate the preamble format and fit the overall style of the eRegs N&C site. 

Here is a rough wireframe of what this may look like. 
screen shot 2016-03-22 at 10 35 13 am

### PR DESCRIPTION
**Pull request by donjo**
This PR provides the base styling for the table of contents to resolve #34.

We're still figuring out the hierarchy here and how section headers should be bolded so this is only covering colors/fonts and not necessarily structure. 

screen shot 2016-03-29 at 3 38 38 pm

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,6 @@ compiled/
 bin/
 include/
 src/
+
+# OSX
+.DS_Store
```
Comment (jmcarp) [inline .gitignore]: Recommend putting these into a global .gitignore and not adding to each repo: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.

Comment (cmc333333) [inline .gitignore]: Doesn't hurt, though

Comment (jmcarp) [inline .gitignore]: Definitely doesn't matter much either way, but if you follow that point to the end, then every dev checks in ignore rules for their OS, editor, plugins, etc., for every project. Which IMO is kind of silly.

**Comment by cmc333333** [comment]
Green means go!

**Pull request merged**